libmscore/drumset.cpp

```diff
@@ -64,6 +71,12 @@ bool Drumset::readProperties(XmlReader& e, int pitch)
       const QStringRef& tag(e.name());
       if (tag == "head")
             _drum[pitch].notehead = NoteHead::name2group(e.readElementText());
+      else if (tag == "noteheads") {
+            while (e.readNextStartElement()) {
+                   const QStringRef& tag(e.name());
+                   _drum[pitch].noteheads[int(NoteHead::name2type(tag.toString()))] = Sym::name2id(e.readElementText());
```

Comment:
A bit dangerous for the changes in future formats, may lead to crash. I would add index boundary check before. I will add within a weekend.

Reply:
Sure. It's rough code for the moment.